src/components/transactions/VotingInfo.vue

```diff
@@ -82,10 +85,48 @@ export default {
       this.setVotesFromFedMembers()
     },
   },
-  created() {
-    this.setVotesFromFedMembers()
+  async created() {
+    try {
+      await this.setVotesFromFedMembers()
+    } catch (error) {
+      console.error(error)
+    }
   },
   methods: {
+    getParamsByTokenType(decodedEvent, event) {
+      switch (this.globalState.currentTokenType) {
+        case TOKEN_TYPE_ERC_20:
+          return [
+            decodedEvent._tokenAddress,
+            decodedEvent._from,
+            decodedEvent._to,
+            decodedEvent._amount,
+            event.blockHash,
+            event.transactionHash,
+            event.logIndex,
+          ]
+        case TOKEN_TYPE_ERC_721:
+          return [
+            decodedEvent._originalTokenAddress,
+            decodedEvent._from,
+            decodedEvent._to,
+            decodedEvent._tokenId,
```

Comment:
as the only diference is the decodedEvent._tokenId, instead of using a switch its easier to create only once the variable and then change the value if it's TOKEN_TYPE_ERC_721
If the diference between them was more than one filed the case then would be better
```js
getParamsByTokenType(decodedEvent, event) {
return [
            decodedEvent._tokenAddress,
            decodedEvent._from,
            decodedEvent._to,
            (this.globalState.currentTokenType === TOKEN_TYPE_ERC_721) ? decodedEvent._tokenId : decodedEvent._amount,
            event.blockHash,
            event.transactionHash,
            event.logIndex,
          ]
    }
```